haystack/document_stores/elasticsearch.py

```diff
@@ -495,11 +114,46 @@ def _create_label_index(self, index_name: str, headers: Optional[Dict[str, str]]
             if not self._index_exists(index_name, headers=headers):
                 raise e
 
+    def _get_raw_similarity_score(self, score):
+        return score - 1000
```

Comment:
Magic numbers are a code smell, what does `1000` mean? Why this exact number? What if I change it?